frontend/public/models/index.ts

```diff
@@ -14,6 +14,19 @@ export const CatalogSourceModel: K8sKind = {
   plural: 'catalogsources',
 };
 
+export const PackageManifestModel: K8sKind = {
+  kind: 'PackageManifest',
+  label: 'PackageManifest',
+  labelPlural: 'PackageManifests',
```

Comment:
We're no entirely consistent on labels, but should this be "Package Manifest" and "Package Manifests"?